packages/api/amplify_api/test/dart_graphql_test.dart

```diff
@@ -218,7 +279,7 @@ void main() {
     });
 
     test('canceled mutation request should never resolve', () async {
-      final req = GraphQLRequest(document: '', variables: {});
+      final req = GraphQLRequest<String>(document: '', variables: {});
```

Comment:
These should still work - was this just to prevent implicit `dynamic`? If so, can we have some tests with explicit `dynamic` if we don't elsewhere, e.g. `GraphQLRequest<dynamic>(...)`

Or better yet, add a task to remove the need for this once and for all 😂

Reply:
It's for implicit `dynamic` like you thought so I will just add a test for explicit dynamic query.